src/tokens-dist/css/variables.css

```diff
@@ -498,9 +676,53 @@
   --eds-theme-color-background-grade-revise-default: var(--eds-color-other-yellow-400); /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
   --eds-theme-color-background-grade-complete-subtle: var(--eds-color-other-mint-100); /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
   --eds-theme-color-background-grade-complete-default: var(--eds-color-other-mint-600); /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
-  --eds-theme-color-background-utility-error: var(--eds-color-other-ruby-100);
-  --eds-theme-color-background-utility-warning: var(--eds-color-other-orange-100);
-  --eds-theme-color-background-utility-success: var(--eds-color-other-mint-100);
+  --eds-theme-color-background-utility-error: var(--eds-color-other-ruby-100); /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
+  --eds-theme-color-background-utility-success: var(--eds-color-other-mint-100); /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
+  --eds-theme-color-background-utility-inverse-high-emphasis-active: var(--eds-color-neutral-100);
```

Comment:
interesting that style-dictionary will  convert `camelCase` to `camel-case` in token names by default (or by our own configuration).